backend/app/controllers/spree/admin/stock_transfers_controller.rb

```diff
@@ -99,16 +100,16 @@ def authorize_transfer_attributes!
         authorize! :create, duplicate
       end
 
-      def transferable_stock_locations
-        Spree::StockLocation.accessible_by(current_ability, :transfer)
+      def load_transferable_stock_locations
+        @stock_locations = accessible_source_stock_locations | accessible_destination_stock_locations
       end
 
-      def load_transferable_stock_locations
-        @stock_locations = transferable_stock_locations
+      def load_source_stock_locations
+        @stock_locations = accessible_source_stock_locations
```

Comment:
would you be opposed to updating this ivar to be `@source_stock_locations` and in the view? was thrown by this a bit.